test/functional/change_stream_spec.test.js

```diff
@@ -156,7 +170,9 @@ describe('Change Stream Spec', function() {
               `Expected there to be an APM event at index ${idx}, but there was none`
             );
           }
-
+          if (events[idx].commandName === 'killCursors') {
```

Comment:
can you call out why we are skipping `killCursors` so future readers don't have the same question I do?

Reply:
I'm not 100% sure about the "why" here, other than that it was specifically called out in the spec changes [here](https://github.com/mongodb/specifications/commit/f93d78191f3db2898a59013a7ed5650352ef6da8#diff-d4053e57c104db86756af1f322023a62R134).

Reply:
I think its enough to link to the spec in a comment here, just a reminder because otherwise it looks like a bandaid fix